doc.go

```diff
@@ -7,13 +7,17 @@ https://github.com/veraison/veraison/tree/main/docs/api
 
 Challenge-Response
 
-The user provides the Evidence generation logics by implementing a
-ChallengeResponseCallback function:
+The user provides the Evidence generation logics by implementing the
+EvidenceBuilder interface:
 
-	func MyChallengeResponseCallback(nonce []byte, accept []string) ([]byte, string, error) {
-		for ct := range accept {
+    type MyEvidenceBuilder struct {
+		// build context (e.g., signing key material, claims template, etc.)
+	}
+
+	func (eb MyEvidenceBuilder) BuildEvidence(nonce []byte, accept []string) ([]byte, string, error) {
+		for _, ct := range accept {
 			if ct == "application/my-evidence-media-type" {
```

Comment:
Perhaps needs to define the media type? What is my-evidence-media-type?

Reply:
It is whatever MIME type you are able to produce.